boards/arm/nucleo_l496zg/pinmux.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2018 Ioannis Konstantelias <ikonstadel@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <device.h>
+#include <init.h>
+#include <pinmux.h>
+#include <sys_io.h>
+
+#include <pinmux/stm32/pinmux_stm32.h>
+
+/* pin assignments for NUCLEO-L496ZG board */
+static const struct pin_config pinconf[] = {
```

Comment:
Maybe you could add at least the pinmux for the UART console. Otherwise most Zephyr examples will not be very useful.

Reply:
@vaussard Console port would be my next step.